vendor/github.com/kubeedge/viaduct

```diff
@@ -1 +0,0 @@
-../../../staging/src/github.com/kubeedge/viaduct/
```

Comment:
And you should not delete this file and other two files for `beehive` and `mapper-framework`, they are link to staging dirs.